plugins/dell-dock2/fu-dell-dock2-wtpd.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2024 Dell Technologies
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-dell-dock2-common.h"
+
+struct _FuDellDock2Wtpd {
+	FuDevice parent_instance;
+};
+
+G_DEFINE_TYPE(FuDellDock2Wtpd, fu_dell_dock2_wtpd, FU_TYPE_DEVICE)
+
+static gchar *
+fu_dell_dock2_wtpd_convert_version(FuDevice *device, guint64 version_raw)
+{
+	return fu_hex_version_from_uint32(version_raw, fu_device_get_version_format(device));
+}
+
+static gboolean
+fu_dell_dock2_wtpd_setup(FuDevice *device, GError **error)
+{
+	FuDevice *proxy = fu_device_get_proxy(device);
+	g_autofree const gchar *devname = NULL;
```

Comment:
Move this to the end of defclarations too.
```suggestion
	g_autofree gchar *devname = NULL;
```

Reply:
Fixed, thanks.